common-lib/common/panels/requests/stat/rate.libsonnet

```diff
@@ -0,0 +1,16 @@
+local g = import '../../../g.libsonnet';
+local generic = import '../../generic/stat/main.libsonnet';
+local base = import './base.libsonnet';
+local stat = g.panel.stat;
+// Uptime panel. expects duration in seconds as input
+base {
+  new(title='Rate', targets, description='Rate of requests.'):
+    super.new(title, targets, description)
+    + self.stylize(),
+
+  stylize(allLayers=true):
+    (if allLayers then super.stylize() else {})
+    + stat.standardOptions.color.withMode('fixed')
+    + stat.standardOptions.color.withFixedColor('light-purple')
```

Comment:
Tiniest nitpick I've ever done, but as a colourblind person I can't really tell the difference between the light purple used here and the blue used in the response time panel. 

Not sure what the solution is, but just wanted to call it out :)

Reply:
Not sure if it helps, changed it to 'light-blue' for duration.

Reply:
Worth a shot, thank you :D